protostar/cheatable_starknet/callable_hint_locals/deploy_contract_hint_local.py

```diff
@@ -19,7 +20,7 @@
 class DeployContractHintLocal(CallableHintLocal):
     def __init__(
         self,
-        declare_cheatcode: DeclareHintLocal,
+        declare_cheatcode: DeclareCairo0HintLocal,
         prepare_cheatcode: PrepareHintLocal,
         deploy_cheatcode: DeployHintLocal,
```

Comment:
why you didn't change the names for these two as well?

Reply:
I can do this in this one but i suppose this would create more noise and i wanted to go one task at a time

Reply:
In such a case, it's good to express such an intention in a task :P I can see you created one after this comment https://github.com/software-mansion/protostar/issues/1645
Thanks, resolving.